cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -149,3 +150,87 @@ def test_new_report_fields(appliance, based_on, request):
     report = appliance.collections.reports.create(**data)
     request.addfinalizer(report.delete_if_exists)
     assert report.exists
+
+
+@pytest.mark.tier(1)
+def test_report_edit_secondary_display_filter(appliance, request, soft_assert):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/6h
+        setup:
+            1. Create/Copy a report with secondary (display) filter.
+        testSteps:
+            1. Edit the secondary filter and test if the report was updated.
+        expectedResults:
+            1. Secondary filter must be editable and it must be updated.
+
+    Bugzilla:
+        1565171
+    """
+    report_data = {
```

Comment:
_Optional_ I generally prefer `data` to be defined outside of the test case as a module constant, but that is mostly just preference.